Classes/ViewHelpers/Pipe/EmailViewHelper.php

```diff
@@ -29,19 +30,12 @@ public function initializeArguments()
     {
         parent::initializeArguments();
         $this->registerArgument('body', 'string', 'Message body. Can also be inserted as tag content');
+        $this->registerArgument('bodySection', 'string', 'Section to use for the body');
         $this->registerArgument('subject', 'string', 'Message subject', true);
-        $this->registerArgument(
-            'recipient',
-            'string',
-            'Message recipient address or name+address as "Name <add@ress>"',
-            true
-        );
-        $this->registerArgument(
-            'sender',
-            'string',
-            'Message sender address or name+address as "Name <add@ress>"',
-            true
-        );
+        $this->registerArgument('recipient', 'string', 'Message recipient address or name+address as "Name <add@ress>"',
+            true);
```

Comment:
This looks odd. Probably needs to be formatted to one argument per line (like left side of diff).